### ISSUE TITLE
System.Globalization.CultureInfo.GetFormat() method should be virtual

### ISSUE THREAD

**Issue by stopiccot** [bridgedotnet/Bridge#3290]
A description of the issue.

### Steps To Reproduce

https://deck.net/8fd24b02e57f7a0d8b67d42a4adc72fe

```csharp
using System;
using System.Globalization;

public class CultureInfoAdapter : CultureInfo {
    public CultureInfoAdapter(string name) : base(name) {
        
    }
    
    public override object GetFormat(Type formatType) {
        return null;
    }
}

public class Program
{
    public static void Main()
    {
        // Place code required to reproduce here
    }
}
```

Code should compile

**Comment by geoffreymcgill** [comment]
Hi @stopiccot 

Thanks for pointing out this defect. A pull-request has been submitted with the fix and should be included in the 16.8.2 release of Bridge. 

Hope this helps.

**Comment by fabriciomurta** [comment]
The implementation looks alright and has been merged to our integration branch. It will make it to the next Bridge release.

**Comment by geoffreymcgill** [comment]
Fixed and included in the Bridge 16.8.2 release.
